sumatra/commands.py

```diff
@@ -126,7 +126,7 @@ def init(argv):
                             description=description)
     parser.add_argument('project_name', metavar='NAME', help="a short name for the project; should not contain spaces.")
     parser.add_argument('-d', '--datapath', metavar='PATH', default='./Data', help="set the path to the directory in which smt will search for output datafiles generated by the simulation/analysis. Defaults to %(default)s.")
-    parser.add_argument('-i', '--input', metavar='PATH', default='/', help="set the path to the directory relative to which input datafile paths will be given. Defaults to the filesystem root.")
+    parser.add_argument('-i', '--input', metavar='PATH', default='.', help="set the path to the directory relative to which input datafile paths will be given. Defaults to the filesystem root.")
```

Comment:
Is this change necessary for portability? 

Having the global root as the default root for the input data store avoids problems in the common case that input data are stored separately from code.